hieradata_aws/production.yaml

```diff
@@ -172,6 +172,8 @@ govuk::apps::service_manual_publisher::ensure: 'present'
 govuk::apps::short_url_manager::ensure: 'present'
 govuk::apps::specialist_publisher::ensure: 'present'
 govuk::apps::travel_advice_publisher::ensure: 'present'
+govuk::apps::design_principles::ensure: 'absent'
+govuk::apps::spotlight::ensure: 'absent'
```

Comment:
I've no idea if this will work 🤷 but it seems like it might!

I would suggest testing this by:

- Loading up the Kibana page for [spotlight healthchecks](https://kibana.logit.io/s/d4dba58d-d7be-4bca-81e7-034e14b8b9ee/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-1h,mode:quick,to:now))&_a=(columns:!(request,status),index:'*-*',interval:auto,query:(language:lucene,query:'request:%22GET%20%2F_healthcheck-ready_spotlight%20HTTP%2F1.1%22'),sort:!('@timestamp',desc))) and the [designprinciple healthchecks](https://kibana.logit.io/s/d4dba58d-d7be-4bca-81e7-034e14b8b9ee/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-1h,mode:quick,to:now))&_a=(columns:!(request,status),index:'*-*',interval:auto,query:(language:lucene,query:'request:%22GET%20%2F_healthcheck-ready_designprinciples%20HTTP%2F1.1%22'),sort:!('@timestamp',desc)))
- [Building your branch of Puppet](https://deploy.integration.publishing.service.gov.uk/job/Deploy_Puppet/)
- Keeping an eye on Kibana and seeing if it has stopped the requests